src/states/adult/tests.rs

```diff
@@ -211,7 +218,7 @@ fn genesis_update_message_successful_trust_check() {
     let gen_pfx_info = adult_test.gen_pfx_info(1);
     let msg = adult_test.genesis_update_message(gen_pfx_info);
 
-    let _ = unwrap!(adult_test.adult.handle_signed_message(msg));
+    let _ = unwrap!(adult_test.adult.handle_message(None, msg, &mut ()));
```

Comment:
I'm a bit confused about why this test function and the next call different functions.